### DISCUSSION TITLE
Welcome to hacktoberfest 2022 Discussions!

### DISCUSSION THREAD

**Discussion by fineanmol** [fineanmol/hacktoberfest#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LucianeFernandesRoque** [comment]
Please I have a question, my pull request was merged but on the site hacktoberfest , could I have done something wrong?

![Screenshot from 2022-10-13 13-59-56](https://user-images.githubusercontent.com/65911301/195691572-7c0bc982-82af-43dc-80a7-5f653fb05e5f.png)
